Support for raspberry pi and new topic for MQTT

vinodsr opened the pull request:
This adds support for raspberry pi. I had added a seperate docker file "Dockerfile.rpi" for raspberry pi build. 

Also added a new MQTT topic  "frigate/<camera_name>/<object_name>/event" Which publishes the image in base64 as well as the event details. This is useful while doing automation events where i need to avoid i am not able to relate the snapshot with the image.

vinodsr committed:
Added docker file for raspberry pi

vinodsr committed:
Added new topic which publishes image as base64 with meta data about the event

vinodsr committed:
Updated the topics in the README (new event)

vinodsr committed:
Merge remote-tracking branch 'upstream/master'

blakeblackshear commented:
Really appreciate the effort on this. I am working to add raspberry pi support in the next release and this will definitely help me get there faster. I have already removed pyarrow as a dependency which will make ongoing support much easier to manage.

vinodsr commented:
Please consider Thant single topic for image and data. It helps in the home automation .

vinodsr committed:
Added support for getting status in the snapshot method.

Need to find a better solution. Current implementation is good wrt the framework

vinodsr committed:
Merge remote-tracking branch 'upstream/master'
